### PR TITLE
Cards test e2e

### PR DESCRIPTION
**Pull request by abelgustavo**

### CODE REVIEW

**Review by carledriss — DISMISSED** [review]
Please resolve conflicts

File: e2e/pages/card.po.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {by} from 'protractor';
+import {CommonActions} from '../utils/CommonActions';
+
+export class Card {
+    URL: string;
+
+    locatorAddAnotherCardLink = by.css('.open-card-composer.js-open-card-composer');
```
Comment (EotronRM) [inline e2e/pages/card.po.ts]: remove 'locator' prefix

File: e2e/pages/card.po.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    locatorTitleTextAreaInput = by.css('.list-card-composer-textarea.js-card-title');
+    locatorAddCardButton = by.css('.js-add-card');
+
+    async createCard(title: string) {
+
```
Comment (EotronRM) [inline e2e/pages/card.po.ts]: delete this blank space

File: e2e/pages/selectedboard.po.ts [hunk 3]
```diff
@@ -12,7 +13,7 @@ export class Selectedboard {
     menuMoreCloseBoard = by.css('.js-close-board');
     confirmCloseBoard = by.css('.js-confirm.full.negate');
     permanentlyDeleteBoardLink = by.css('.quiet.js-delete');
-
+    AddAnotherCardLink = by.css('.open-card-composer.js-open-card-composer');
```
Comment (EotronRM) [inline e2e/pages/selectedboard.po.ts]: camelCase

File: e2e/specs/card.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import {Dashboard} from '../pages/dashboard.po';
+import {Login} from '../pages/login.po';
+import {Card} from '../pages/card.po';
+import {Selectedboard} from '../pages/selectedboard.po';
+import user from '../utils/environment.json';
+
+describe('', () => {
```
Comment (EotronRM) [inline e2e/specs/card.spec.ts]: give a description in this spec

File: e2e/specs/card.spec.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    let card: Card;
+    const url = 'https://trello.com/login?returnUrl=%2F';
+    beforeEach(async () => {
+        login = new Login(url);
+        await login.loadPage();
+        board = await login.LogInWithUser(user.owner1);
+        selectedBoard = await board.selectDashBoard('ae');
+    });
+
+    it('When user selected a one  board, he can to create a new card', async () => {
```
Comment (EotronRM) [inline e2e/specs/card.spec.ts]: All the specs will be in first-person

File: protractor.conf.ts [hunk 6]
```diff
@@ -6,15 +6,16 @@ export let config: Config = {
         browserName: 'chrome',
     },
     specs: [
-        './e2e/specs/createdashboard.spec.js',
+        './e2e/specs/card.spec.js',
     ],
     seleniumAddress: 'http://localhost:4444/wd/hub',
+    // SELENIUM_PROMISE_MANAGER: false,
```
Comment (EotronRM) [inline protractor.conf.ts]: Don't modify the protractor conf! let like the develop

File: protractor.conf.ts [hunk 7]
```diff
@@ -17,4 +17,4 @@ export let config: Config = {
     jasmineNodeOpts: {
         defaultTimeoutInterval: 120000,
     },
-};
+};
```
Comment (EotronRM) [inline protractor.conf.ts]: space before this

File: e2e/pages/card.po.ts [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import {by} from 'protractor';
+import {CommonActions} from '../utils/CommonActions';
+
+export class Card {
+    URL: string;
```
Comment (EotronRM) [inline e2e/pages/card.po.ts]: delete this

**Review by EotronRM — APPROVED** [review]

**Pull request merged**